src/sidebar-counters/sidebar-counters.js

```diff
@@ -4,66 +4,80 @@
 import { h } from 'dom-chef'
 import elementReady from 'element-ready'
 import api from '../api'
+import SelectorObserver from 'selector-observer'
 
 import './sidebar-counters.css'
 
-export const addBadge = (
-    a: HTMLAnchorElement,
-    resources: {| size: number |} | void
-) => {
-    const navLinksContainer: HTMLElement = (a.parentNode: any)
-    navLinksContainer.style.overflow = 'hidden'
+const HREF_BRANCHES = 'branches'
+const HREF_PULL_REQUESTS = 'pull-requests'
+export const menus = [HREF_BRANCHES, HREF_PULL_REQUESTS]
 
-    a.style.position = 'relative'
+const MAX_COUNTER = 99
 
-    const size =
-        // eslint-disable-next-line eqeqeq, no-eq-null
-        resources == null || resources.size == null
-            ? '?'
-            : resources.size > 999999
-                ? '999999+'
-                : resources.size
+export let menusCounters = {
+    HREF_BRANCHES: null,
+    HREF_PULL_REQUESTS: null,
+}
+
+type ResultSize = $Exact<{ size: number }>
 
-    const badge = (
+export function getBadge(size: number): HTMLElement {
+    const maxReached = size > MAX_COUNTER
+    return (
         <span class="__rbb-badge">
-            <span class="__rbb-badge-counter">{size}</span>
+            <span class={`__rbb-badge-counter${maxReached ? ' -max' : ''}`}>
```

Comment:
Can we rename the `max` class to the one below, and change this expression to

```suggestion
            <span class={maxReached ? '__rbb-badge-counter-max` : '__rbb-badge-counter'}>
```

So that it passes the [grep test](http://jamie-wong.com/2013/07/12/grep-test/)